spec/services/linked_data/mapper/term_results_mapper_service_spec.rb

```diff
@@ -2,56 +2,93 @@
 
 RSpec.describe Qa::LinkedData::Mapper::TermResultsMapperService do
   let(:graph) { Qa::LinkedData::GraphService.load_graph(url: 'http://local.data') }
-  let(:predicate_map) do
+  let(:subjects) { subject.map { |result| result[:uri].first.to_s } }
+
+  let(:expected) do
```

Comment:
Both the predicate_map test and the ldpath_map test use the same expected results.  Both approaches should give the same results.